partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,86 @@
+        part = project.get_part("input_part")
+        if not part:
+            raise RuntimeError("Failed to load the input part.")
+
+        # Export the part to the desired output format
+        export_part(part, output_filename, output_type, ctx)
+    finally:
+        shutil.rmtree(temp_dir)  # Cleanup temporary files
+
+
+
+def generate_partcad_config(temp_dir, input_type, temp_input_path):
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_partcad_config`

Add return type annotation: `None`

(ANN201)